.github/workflows/build-be.yml

```
@@ -160,28 +160,18 @@ jobs:
           echo "doorstop_version=$VERSION" >> $GITHUB_ENV
       - name: Create zips
         run: |
+          mkdir dist
           cd artifacts
-          (cd doorstop_win_release; zip -r ../doorstop_win_release_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_win_verbose; zip -r ../doorstop_win_verbose_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_linux_release; zip -r ../doorstop_linux_release_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_linux_verbose; zip -r ../doorstop_linux_verbose_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_linux_debug; zip -r ../doorstop_linux_debug_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_macos_release; zip -r ../doorstop_macos_release_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_macos_verbose; zip -r ../doorstop_macos_verbose_${{ env.doorstop_version }}.zip *)
-          (cd doorstop_macos_debug; zip -r ../doorstop_macos_debug_${{ env.doorstop_version }}.zip *)
+          zip -r ../dist/doorstop_all_in_one_${{ env.doorstop_version }}.zip *
```

Maybe some "shorter" name?
```suggestion
          zip -r ../dist/doorstop_all_${{ env.doorstop_version }}.zip *
```